internal/pkg/service/buffer/task/orchestrator/orchestrator_test.go

```diff
@@ -102,14 +102,14 @@ func TestOrchestrator(t *testing.T) {
 	// Put some key to trigger the task
 	assert.NoError(t, pfx.Key("key1").Put(testResource{ReceiverKey: receiverKey, ID: "ResourceID"}).Do(ctx, client))
 
-	// Wait for task on the node 1
+	// Wait for task on the node 2
 	assert.Eventually(t, func() bool {
-		return strings.Contains(d1.DebugLogger().AllMessages(), "DEBUG  lock released")
+		return strings.Contains(d2.DebugLogger().AllMessages(), "DEBUG  lock released")
```

Comment:
V tom testu se nějak záhadně vyměnilo který node task zpracoval, nevím proč. 😅

Reply:
O tom, ktory node si to vezme rozhoduje tato cast: 
https://github.com/keboola/keboola-as-code/blob/2b9d0a77168083a6a2e0de54b7d96f3a868a6925/internal/pkg/service/buffer/task/orchestrator/orchestrator_test.go#L72-L73

A ten string sa zmenil, uz tam nebude napr. `.../00001000/...`. ale `.../1000/...`.